### ISSUE TITLE
What's supposed to be in the PATH?

### ISSUE THREAD

**Issue by Jazzman123** [aspnet/Docs#11065]
The explanation for "No .NET Core SDKs were detected" is not very helpful.  What exactly is supposed to be in the path?  What single folder represents "the SDK"?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 967496ea-59b6-1f17-4f39-1e4c232126c6
* Version Independent ID: 09bb2010-fe5c-1061-23ec-a0dcc92bdd3f
* Content: [Troubleshoot ASP.NET Core projects](https://docs.microsoft.com/en-us/aspnet/core/test/troubleshoot?view=aspnetcore-2.2#no-net-core-sdks-were-detected)
* Content Source: [aspnetcore/test/troubleshoot.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/test/troubleshoot.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by bryanfarrell** [comment]
It's good that you guys updated the documentation to say what should be in the path.  It would also be nice if you replicated the problem as I followed the instructions and made sure the PATH is set correctly and I still get the error after installing the latest .NET Core version.

**Comment by guardrex** [comment]
@bryanfarrell Open an issue for engineering on the aspnet/AspNetCore issues. We can't troubleshoot  app problems here on the docs repo. In case this is related to installing the Hosting Bundle that I just added some content on that scenario (it's in review): https://github.com/aspnet/Docs/pull/11386

**Comment by bryanfarrell** [comment]
@guardrex
> In case this is related to installing the Hosting Bundle that I just added some content on that scenario (it's in review): #11386

Thanks for the quick response.  After some more searching on a solution I found out it is related to the Hosting Bundle install.  I found this closed issue [#4350](https://github.com/dotnet/core-setup/issues/4350) under dotnet core-setup that showed that installing Hosting Bundle installs both x86 and x64 Core Runtimes.  I uninstalled the x86 version and Visual Studio found the SDK's after doing so.  Thanks again for the help.
